addons/event/views/event_views.xml

```diff
@@ -232,8 +232,7 @@
                                     <tree string="Communication" editable="bottom">
                                         <field name="sequence" widget="handle"/>
                                         <field name="notification_type"/>
```

Comment:
This would be thrown out of the front-end too, and likely could be removed on master